catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -536,27 +525,53 @@ public void run() {
 
         public CallableSourceResponse(Source source, QueryRequest request) {
             this.source = source;
-            this.request = request;
+            this.request = new QueryRequestImpl(request.getQuery(), new HashMap<>(request.getProperties()));
         }
 
         @Override
         public SourceResponse call() throws Exception {
```

Comment:
This should be annotated with `@Nullable` since we can return null from this method.